test/src/test/resources/fixtures/namespaced/typename-is-sent-to-owning-service-even-when-no-fields-are-queried.yml

```diff
@@ -0,0 +1,199 @@
+name: typename is sent to owning service even when no fields are queried
```

Comment:
Added a test where the query doesn't contain any fields from the service that owns the namespaced field

Reply:
I think would be good to have a test with the same scenario that surfaced the bug. The namespaced type is extended in the same service where it's defined. What do you think?

Reply:
Yep having both this test and the one @temaEmelyan suggested would be great.